src/EnergyPlus/RefrigeratedCase.cc

```diff
@@ -7725,77 +7723,78 @@ void SetupReportInput(EnergyPlusData &state)
         // Setup Report Variables for simulated  Walk In (do not report unused WalkIns)
         // CurrentModuleObject='Refrigeration:WalkIn'
         for (int walkInNum = 1; walkInNum <= state.dataRefrigCase->NumSimulationWalkIns; ++walkInNum) {
-            if (WalkIn(walkInNum).NumSysAttach == 1) { // ensure no unuseds reported
+            auto &walkin = WalkIn(walkInNum);
```

Comment:
What's the logic behind using `thisCase` sometimes and `walkin` other times?

Reply:
Well, my typical approach has been `this*`, but in many cases, that it because I always hope to revisit those functions and just swap them to methods and use `this->` directly.  In the refrigerated case files, there are so many "things", it's a gigantic jungle.  And the responsibilities of each function are enormous, with tons of loops over many different things.  Ultimately, I decided to just name things for what they are to make the code as neat and tidy as possible.  I understand it still needs tons of work in here to make it maintainable, just moving it one step in that direction.